contents/blog/how-to-create-personas.md

```diff
@@ -0,0 +1,148 @@
+
+For example, let's say you're building an email client. You ask your receive what obstacles they face when trying to achieve their goals and here's what they:
+
+
+| Job title | Participant’s answer | Themes |
+|--------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|---------------------|
+| **CTO** | I get distracted by too many non-priority emails that appear at the top of my inbox. | Email overload, prioritization |
+| **CEO** | Notifications don't work properly, so I miss important updates. I end up prioritizing the wrong work | Missed information, prioritization |
+| **Founder** | I feel overwhelmed by the volume of emails I receive daily. It's hard to prioritize tasks. | Email overload, prioritization |
+| **Developer** | The search function in my email client is too slow, making it hard to find old emails. | Slow search |
+| **Head of Product** | Due to the high volume of status updates I get via email, it's challenging to understand the overall project progress. | Email overload, project management |
+
+
+In the above, we see that your core users are **senior leaders** and the two most common themes are **email overload** and **prioritization**. Thus you'll include this in your persona.
+
+### Step 3: Combine your data with product analytics
+
+Now that you have your qualitative data, you can combine it with quantitative data to paint a clear picture of your who your users are. 
+
+Product analytics unlocks key insights into how users that match your persona are using your product, and whether their stated problems and behaviors align with real-world usage.
+
+For example, continuing on our previous example of an email app, we could use product analytics to answer the following questions:
+
+| Theme               | Data questions    |
+|--------------------------------|----------------------------------------------------------------------------------------------------|
+| **Email Overload**             | How many emails do they receive every day?                                                          |
+|                                | How much time do they spend in the app compared to other users?                                    |
+|                                | Which folders or categories have the highest volume of unread emails?                              |
+| **Prioritization**             | What percentage of users who mention "prioritization" use features related to sorting or flagging emails? |
+|                                | Are there features within the product they aren't using that could help address this problem?      |
+|                                | How frequently do these users set reminders or snooze emails?                                      |
+
+To help you brainstorm questions to ask for your own product, here's a list of common insights to dig into:
```

Comment:
What specific insights should they look at here?

- Percentage of users that match a persona filter
- Cohort persona filter
- Feature usage with person filter
- User path with persona filter

Maybe it is worth mentioning it is easiest to create a persona filter

Reply:
It's not clear to me what you are suggesting, but my understanding is that most of these questions are covered in the list below?

Re "easiest to create a persona filter": Added now.

Reply:
I guess the below points answer these 😄